### PR TITLE
remove spammy no members available warning

### PR DESCRIPTION
**Pull request by mightyguava**
This is an expected state in all 1 pod clusters and on first service
launch.

### CODE REVIEW

File: misk-zookeeper/src/main/kotlin/misk/clustering/zookeeper/ZkLease.kt [hunk 1]
```diff
@@ -201,7 +201,7 @@ internal class ZkLease(
     val desiredLeaseOwner = try {
       clusterSnapshot.resourceMapper[leaseResourceName]
     } catch (e: NoMembersAvailableException) {
-      log.warn { e.message }
+      // no healthy members in the cluster yet
```
Comment (ryanhall07) [inline misk-zookeeper/src/main/kotlin/misk/clustering/zookeeper/ZkLease.kt]: can we instead add `tryResourceMapper` method that returns a nullable owner?

Comment (mightyguava) [reply misk-zookeeper/src/main/kotlin/misk/clustering/zookeeper/ZkLease.kt]: sure, I added it to this class, didn't want to change the interface

Comment (tirsen) [reply misk-zookeeper/src/main/kotlin/misk/clustering/zookeeper/ZkLease.kt]: just so I understand this correctly: what does returning null here mean? does that mean the current process is the owner? or it means there is no owner so it tries to grab the lock?

Comment (mightyguava) [reply misk-zookeeper/src/main/kotlin/misk/clustering/zookeeper/ZkLease.kt]: It means there is no healthy pod to act as the owner, so no one will try to grab the lock

**Pull request merged**